resources/views/photos/album.blade.php

```diff
@@ -66,6 +67,7 @@
     @parent
 
     <script>
+        /*
```

Comment:
Can be deleted, old code is available in version control. :)

Reply:
Removed